src/cmakeTaskProvider.ts

```diff
@@ -326,44 +335,27 @@ export class CustomBuildTaskTerminal implements vscode.Pseudoterminal, proc.Outp
             }
         } else {
             log.debug(localize("cmake.driver.not.found", 'CMake driver not found.'));
-            this.writeEmitter.fire(localize("build.failed", "Build failed.") + endOfLine);
+            this.writeEmitter.fire(localize("task.failed", "{0} failed.", taskName) + endOfLine);
```

Comment:
Don't rename the localization ids for now.  They don't actually matter.  If you do rename them, the translations will all reset and we don't want that right before we release.

So please change them back to `"build.failed"`, etc instead of `"task.failed"`